docs/troubleshooting_and_faq/docker.md

```diff
@@ -0,0 +1,13 @@
+# Docker
+
+
+### When starting Docker, I get 'Permission denied while trying to connect to the Docker daemon'
+
+If you're having trouble getting Docker to run on your system, you can refer to the Docker documentation for instructions
+on how to start Docker on [macOS](https://docs.docker.com/docker-for-mac/install/), [Linux](https://docs.docker.com/desktop/install/debian/#launch-docker-desktop), and [Windows 10 and 11](https://docs.docker.com/desktop/install/windows-install/#start-docker-desktop).
+
+### When starting Docker on Windows, I get 'Cannot connect to Docker daemon'
+
+If you're having trouble with permissions on a Windows system, you can refer to the Docker documentation for instructions
+on how to [Understand permission requirements for Windows](https://docs.docker.com/desktop/windows/permission-requirements/).
+
```

Comment:
We may want to link to these from the mp-build and mp-test sections, as that's often when they'll occur

Reply:
I like what you did, but I was actually referring to the troubleshooting sections - since that's what the focus is